scripts/configmap.sh

```
@@ -2,5 +2,4 @@
 NAMESPACE=ddis-asteroid--usernaut-${ENVIRONMENT}
 
 kubectl create cm usernaut-config --from-file=default.yaml=./appconfig/default.yaml \
---from-file=${ENVIRONMENT}.yaml=./appconfig/${ENVIRONMENT}.yaml -n ${NAMESPACE}
-
+--from-file=${ENVIRONMENT}.yaml=./appconfig/${ENVIRONMENT}.yaml -n ${NAMESPACE} --dry-run=client -o yaml | kubectl apply -f -
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)